modin/pandas/test/dataframe/test_udf.py

```diff
@@ -228,8 +228,7 @@ def add(a, b, c):
 def test_apply_udf(data, func):
     eval_general(
         *create_test_dfs(data),
-        lambda df, *args, **kwargs: df.apply(*args, **kwargs),
-        func=func,
+        lambda df, *args, **kwargs: df.apply(func, *args, **kwargs),
```

Comment:
a minor change that allows to get rid of a dirty hack of defining test `func`s as a double lambdas